### ISSUE TITLE
Clarify output for loss.grad_fn

### ISSUE THREAD

**Issue by jlin27** [pytorch/tutorials#784]
In the [Neural Network tutorial](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#sphx-glr-beginner-blitz-neural-networks-tutorial-py), add a note to clarify what object means. The output aren't easily explainable: 
![image](https://user-images.githubusercontent.com/8042156/70949272-38a41900-2012-11ea-8796-9d3acb509cd3.png)

**Comment by jlin27** [comment]
Addressed in PR #922
